# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: 87f3532bfbcd
# Full Hash: 87f3532bfbcd50706f16c61eff4c6bdd93fe34aa
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-15 18:16:35
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Share all-of-dcomp.h preamble, and deal with outdated mingw dcomp.h. r=gfx-reviewers,sotaro
#   
#   Mingw's dcomp.h is not the official one, but rather a by-hand
#   reproduction. While this newly-updated version has e.g.
#   IDCompositionFilterEffect, it is still missing e.g.
# ==============================================================================

diff -r 9c1d9405e8bf -r 87f3532bfbcd gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Wed Feb 15 08:29:43 2023 +0000
+++ b/gfx/thebes/moz.build	Wed Feb 15 08:29:43 2023 +0000
@@ -67,6 +67,7 @@
 ]
 
 EXPORTS.mozilla.gfx += [
+    "AllOfDcomp.h",
     "D3D11Checks.h",
     "DeviceManagerDx.h",
     "DisplayConfigWindows.h",